lib/reform/form/validation/unique_validator.rb

```diff
@@ -1,7 +1,14 @@
 class Reform::Form::UniqueValidator < ActiveModel::EachValidator
   def validate_each(form, attribute, value)
-    if form.model.class.where("#{attribute} = ?", value).size > 0
-      form.errors.add attribute, "#{attribute} must be unique."
+    if form.model.persisted?
+      if form.model.class.where("#{attribute} = ? AND id <> ?", value, form.model.id).size > 0
```

Comment:
I think it's better to not interpolate an `attribute` in sql query like this because of possible escaping and quoting problems. So this can be changed to 
```ruby
query = form.model.class.where(attribute => value)
if form.model.persisted?
  query = query.merge(form.model.class.where("id <> ?", form.model.id)
end

if query.any?
    form.errors.add(attribute, "#{attribute} must be unique.")
end
```

What do you think?